package/network/config/netifd/files/usr/libexec/network/packet-steering.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/sh
+
 NPROCS="$(grep -c "^processor.*:" /proc/cpuinfo)"
 [ "$NPROCS" -gt 1 ] || exit
 
-PROC_MASK="$(( (1 << $NPROCS) - 1 ))"
+PROC_MASK="$(( (1 << NPROCS) - 1 ))"
 
 find_irq_cpu() {
-	local dev="$1"
-	local match="$(grep -m 1 "$dev\$" /proc/interrupts)"
-	local cpu=0
+        dev="$1"
```

Comment:
why are we mixing space and tab for indent???

Reply:
Corrected indents